sycl/include/CL/sycl/handler.hpp

```diff
@@ -739,11 +739,29 @@ class __SYCL_EXPORT handler {
     !defined(DPCPP_HOST_DEVICE_OPENMP) &&                                      \
     !defined(DPCPP_HOST_DEVICE_PERF_NATIVE) && SYCL_LANGUAGE_VERSION >= 202001
     // Range should be a multiple of this for reasonable performance.
-    constexpr size_t MinFactorX = 16;
+    size_t MinFactorX = 16;
     // Range should be a multiple of this for improved performance.
-    constexpr size_t GoodFactorX = 32;
+    size_t GoodFactorX = 32;
     // Range should be at least this to make rounding worthwhile.
-    constexpr size_t MinRangeX = 1024;
+    size_t MinRangeX = 1024;
+
+    // Parse optional parameters of this form:
+    // MinRound:PreferredRound:MinRange
+    char *RoundParams = getenv("SYCL_PARALLEL_FOR_RANGE_ROUNDING_PARAMS");
```

Comment:
Can we use SYCLConfig interface here (e.g. SYCLConfig<SYCL_DEVICE_FILTER>)?
In any case it is resonable to add environment variable to sycl/doc/EnvironmentVariables.md

Reply:
This feature is supposed to improve out-of-the-box experience with SYCL parallel_for. It is not meant to be exposed to users. So I have added env var control only for internal experimentation. It should not be documented.